app/forms/referrals/teacher_role/employment_status_form.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+module Referrals
+  module TeacherRole
+    class EmploymentStatusForm
+      include ActiveModel::Model
+      include ValidatedDate
+
+      attr_accessor :referral,
+                    :role_end_date,
+                    :employment_status,
+                    :reason_leaving_role
+
+      validates :referral, presence: true
+      validates :employment_status,
+                inclusion: {
+                  in: %w[employed suspended left_role]
+                }
+      validates :reason_leaving_role,
+                inclusion: {
+                  in: %w[resigned dismissed retired unknown]
+                },
+                if: -> { left_role? }
+
+      def save(params = {})
```

Comment:
```suggestion
      def save(date_params: {})
```

I think we can be more explicit here, until we have good reason otherwise.